src/modules/nameSchema/docs/de_DE.md

```diff
@@ -0,0 +1,65 @@
+
+| Parameter        | Bedeutung                                                                                                                              | Beispiel                           | Standard   |
+|------------------|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|------------|
+| `padding`        | Anzahl der Stellen, die die Nummerierung haben soll. Führende Nullen werden hinzugefügt. Nur sinnvoll mit der Verwendung von `arabic`. | `{{unitIndex:2}}` ergibt `01`      | `0`        |
+| `start`          | Startwert der Nummerierung.                                                                                                            | `{{unitIndex::10}}` ergibt `10`    | `1`        |
+| `numeral system` | System, in dem die Nummerierung erfolgen soll.                                                                                         | `{{unitIndex:::roman}}` ergibt `I` | `arabic`   |
+| `group`          | Gruppierung der Fahrzeuge für die Nummerierung.                                                                                        | `{{unitIndex::::dispatch}}`        | `building` |
+
+#### `unitIndex`
+
+| Template                 | Ergebnis              |
+|--------------------------|-----------------------|
+| `{{unitIndex}}`          | `1`, `2`, `3`, ...    |
+| `{{unitIndex:2}}`        | `01`, `02`, `03`, ... |
+| `{{unitIndex::10}}`      | `10`, `11`, `12`, ... |
+| `{{unitIndex:::roman}}`  | `I`, `II`, `III`, ... |
+| `{{unitIndex:2:8}}`      | `08`, `09`, `10`, ... |
+| `{{unitIndex::5:roman}}` | `V`, `VI`, `VII`, ... |
+| `{{unitIndex::5:alpha}}` | `V`, `VI`, `VII`, ... |
+
+#### `numeral system`
+
+Alle Systeme außer `arabic` unterstützen die Suffixe `-lower` und `-upper` für Klein- und Großschreibung (z.B. `roman-lower`).
+
+| Wert     | Beschreibung                     | Beispiel                         |
+|----------|----------------------------------|----------------------------------|
+| `arabic` | Arabische Zahlen (Standard)      | `1`, `2`, `3`, ...               |
+| `roman`  | Römische Zahlen                  | `I`, `II`, `III`, ...            |
+| `alpha`  | Buchstaben                       | `A`, `B`, `C`, ...               |
+| `greek`  | Griechische Buchstaben (max. 23) | `Alpha`, `Beta`, `Gamma`, ...    |
+| `icao`   | ICAO-Phonetic (max. 25)          | `Alpha`, `Bravo`, `Charlie`, ... |
+| `emoji`  | Emoji Ziffern                    | 0️⃣, 1️⃣2️⃣3️⃣, 6️⃣6️⃣6️⃣, ...   |
+
+#### `group`
+
+| Wert               | Gruppieren nach                                                       | Verwendbar in Templates für |
+|--------------------|-----------------------------------------------------------------------|-----------------------------|
+| `none`             | Ohne Gruppierung, Fahrzeuge werden global durchnummeriert (Standard). | Einheiten, Gebäuden         |
```

Comment:
I am not sure if that is a good idea for standard. In V3, `buildingUnitType` is standard. Maybe we could do a survey on that?

Reply:
Changed the default to `buildingVehicleType`.